### PR TITLE
Test models/22

### PR DESCRIPTION
**Pull request by gaoshh711**
first model test

### CODE REVIEW

File: spec/model/county_model_spec.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe County do
```
[the remark this answers is not in the document]
Comment (gaoshh711) [reply spec/model/county_model_spec.rb:5]: I think it is normal to have be in similar case.

File: spec/model/event_model_spec.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+require 'rails_helper'
+describe Event do
+  describe '.county_names_by_id' do
+    let(:state) { create(:state) }
+    # byebug
+    let(:counties) { create_list(:county, 3, state: state) }
+    let(:event) { create(:event, county: counties.first) }
+    let(:event_with_no_county) { create(:event, county: nil) }
+
+    it 'returns a hash table with normal parameter' do
```
[the remark this answers is not in the document]
Comment (gaoshh711) [reply spec/model/event_model_spec.rb:12]: I think it is normal to have be in similar case.

**Comment by Clarivy** [comment]
The title makes no sense

File: spec/factories/event.rb
Comment (Clarivy) [inline spec/factories/event.rb:1]: Use English please :)

**Review by Clarivy — APPROVED** [review]
Ok. I'll merge it

**Pull request merged**